cli/azd/pkg/project/framework_service_docker_test.go

```diff
@@ -54,17 +56,24 @@ services:
 		})
 
 	mockContext.CommandRunner.When(func(args exec.RunArgs, command string) bool {
-		return strings.Contains(command, "docker build")
+		return strings.Contains(command, "docker build") && !strings.Contains(command, "-q")
```

Comment:
Can we remove this now?  I think we no longer call this with `-q` in any cases (since we are on the `--iidfile` plan), so this should always be true?